client/src/pages/SymptomsCheck.js

```diff
@@ -1,4 +1,9 @@
 import React, { useState } from "react";
+import { Button, Radio } from 'antd';
+import { Link, withRouter } from "react-router-dom";
+import "../styles/style.css"; 
```

Comment:
Probably better to use StyledComponents :) 
you can just copy that css file into  something like 
```
`const MyComponent= styled.p`
  // put css here
`
```

and then use that component in a JSX tag as html element / React compoment...